CONTRIBUTING.md

```diff
@@ -1,65 +1,251 @@
+
+1. [Create a pull request][create-pr]
+   and verify that you have the right target branch.
+   Most PRs are merged into the `main` branch.
+   If you need to target an upcoming release or new feature,
+   your PR might be merged to a dedicated branch with a different name.
+
+1. Write the PR summary using the following guidelines:
+
+   1. Describe the change.
+      Include a brief description of what's changing in the pull request.
+      Do this even if you link to a relevant issue.
+
+   1. Link to any related issues. If the PR should fix a particular issue,
+      include a line that states `Fixes #<issue number>`.
+      (Including this phrase links the PR to the issue.
+      When you close the PR, GitHub closes the issue).
+      If your PR fixes multiple issues,
+      include this phrase in whole for each issue.
+
+   1. Point to specific changes. If your PR is large,
+      consider pointing the reviewer to the specific files or lines
+      you'd like the reviewer to check.
+
+1. If your PR isn't ready for review,
+   [set it to Draft][set-draft] and, if possible,
+   [apply the **st.WIP** label][apply-wip-label].
+
+1. Click **Create pull request**. This starts the review process.
+
+### Respond to reviewer feedback
+
+1. Read and address comments from the reviewer.
+   Acknowledge all comments with at least a [reaction emoji][reaction-emoji].
+   If the changes are technically or grammatically correct, make them.
+   Reply to comments that you want to discuss.
+   Remember that all comments are public. Keep your replies civil and relevant.
+
+1. When your PR is ready for another round of feedback,
+   re-request a review or write a comment with `PTAL @[reviewer]`.
+   (PTAL means _please take another look_.)
+
+1. Once your PR is approved, a reviewer [merges it][merges-pr].
+
+## Reviewers
+
+### General guidelines for reviews
+
+* Technical review comes before content, or "copy", review.
+* Lean toward velocity. Aim for _improvement_ over absolute _perfection_.
+* Check the whole page. Sometimes new content contradicts unchanged
+  content elsewhere on the same page.
+* Prefer giving an overall note when there's a repeated problem,
+  rather than commenting on every instance.
+* Try to **approve** with suggestions wherever possible.
+  Use **request changes** only when the change can harm an end user's
+  work.
+* Close PRs that violate [Flutter's Code of Conduct][code-of-conduct].
+
+### Perform a technical review
+
+In general, follow these practices.
+
+* Critique the _technical aspects_ of the PR.
+  If technical content could be made less complex or more detailed,
+  advise the writer that this is the case in your review.
+  Still _approve the PR_ but only if your changes are included.
+* Provide suggested changes with corrected copy to make your changes clear.
+
+To conduct a consistent technical review in the Flutter website:
+
+1. [Review a PR][review-pr] when it meets the following conditions:
```

Comment:
This is a lot for me to think about. Shouldn't I just review a PR if I'm asked to review it?

Reply:
Right now, GitHub actions set who reviews. The hope will be to remove this automatic setting soon. Because of this automation, a review for a PR is requested even if the author is not done with it. The review request via GitHub has lost its meaning as is.

Reply:
It would be GREAT if we could only add the DRE that is on rotation. The original plan was to use a weekly rotation to reduce the email noise for everyone.

Reply:
The framework team individually tags reviewers. Maybe we can copy that workflow here for the DRE who's on rotation that week?